src/Model/PhasorDynamics/Governor/Tgov1/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+# [[
+#  Author(s):
+#    - Luke Lowery <lukel@tamu.edu>
+#    - Adam Birchfield <abirchfield@tamu.edu>
+# ]]
+
+gridkit_add_library(phasor_dynamics_governortgov1
+  SOURCES
+    Tgov1.cpp
+  OUTPUT_NAME
+    gridkit_phasor_dynamics_governortgov1)
```

For now, make it link directly to `Genrou` model:
```suggestion
gridkit_add_library(phasor_dynamics_governortgov1
  SOURCES
    Tgov1.cpp
  LINK_LIBRARIES
    GRIDKIT::phasor_dynamics_genrou
  OUTPUT_NAME
    gridkit_phasor_dynamics_governortgov1)
```
Otherwise, the code will not build. This should be only a temporary solution until we define signal buses.

---

I agree, Signal buses should be its own PR. I am unsure why my machine raised no build errors. I accept this suggested change.